android/src/main/java/io/ably/flutter/plugin/AblyEventStreamHandler.java

```diff
@@ -125,6 +124,7 @@ public void onListen(Object object, EventChannel.EventSink uiThreadEventSink) {
         final AblyEventMessage<Object> eventMessage = ablyMessage.message;
         final String eventName = eventMessage.eventName;
         try {
+            ChannelOptions channelOptions;
```

Comment:
I'm not sure why this local variable has to sit outside at this scope. There should be an instance of it for each case that uses it, is all.

Reply:
It will be duplicate definition if declared inside each switch case